dark/assets/css/styles.css

```diff
@@ -29,7 +26,12 @@
   color: black;
   background: var(--primary-color);
 }
-
+/*  Utility */

```

Comment:
Add a new line before this @priyankarpal 
```suggestion

/*  Utility */
```

Reply:
Don't resolve the conversation without doing the changes @priyankarpal 
Do these changes first

Reply:
check I deleted my own branch. I change HTML and CSS code to main.